R/loo_compare.R

```diff
@@ -143,12 +167,16 @@ print.compare.loo <- function(x, ..., digits = 1, simplify = TRUE) {
   } else if (NCOL(xcopy) >= 2 && simplify) {
      xcopy <- xcopy[, c("elpd_diff", "se_diff")]
   }
-  print(.fr(xcopy, digits), quote = FALSE)
-  invisible(x)
+  if (p_worse) {
```

Comment:
This will cause an error when printing because the argument name is `pnorm`. But maybe the argument name should be `p_worse`? Either way, either the argument name needs to change or this `if` clause needs to change.